cmd/buck/cli/cli.go

```diff
@@ -80,6 +80,17 @@ func Init(baseCmd *cobra.Command) {
 	archiveCmd.Flags().StringP("file", "f", "", "Optional path to a file containing archive config json that will override the default")
 	archiveCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
 
+	setDefaultArchiveConfigCmd.Flags().IntP("rep-factor", "r", 1, "Target number of active deals")
+	setDefaultArchiveConfigCmd.Flags().Int64P("deal-min-duration", "d", 518400, "Minimum duration for the deal")
+	setDefaultArchiveConfigCmd.Flags().StringSliceP("excluded-miners", "x", nil, "Miner addresses that should not be used")
+	setDefaultArchiveConfigCmd.Flags().StringSliceP("trusted-miners", "t", nil, "Miner addresses that must be used")
+	setDefaultArchiveConfigCmd.Flags().StringSliceP("country-codes", "c", nil, "Select miners with specific countries")
+	setDefaultArchiveConfigCmd.Flags().Uint64P("max-price", "p", 0, "Maximum price that will be spent per RepFactor in AttoFIL/GiB per Epoch")
```

Comment:
what to set this now?

Reply:
I'll move the semantics to "update" rather than set.
I'll continue to work it on this branch.